packages/@atjson/document/src/index.ts

```diff
@@ -1,29 +1,51 @@
-import Annotation from './annotation';
+import Annotation, { AnnotationConstructor, AnyAnnotation } from './annotation';
+import { Block, Inline, Object, Parse, Unknown } from './annotations';
+import { Attribute, Attributes } from './attributes';
+import Change, { AdjacentBoundaryBehaviour, Deletion, Insertion } from './change';
+import JSON from './json';
 import Query, { Filter, flatten } from './query';
-import Schema, { Display } from './schema';
 
-const OBJECT_REPLACEMENT = '\uFFFC';
+export interface AnnotationJSON {
+  type: string;
+  start: number;
+  end: number;
+  attributes: Attributes;
+}
 
-export { Annotation, Schema, Display };
```

Comment:
A few places still try to [import Display](https://github.com/CondeNast-Copilot/atjson/blob/2b5b08e6f2a09bbcb1ffd7995a7a72ff4484849c/packages/%40atjson/schema/src/index.ts#L1); want I should make a PR against this to fix?

Reply:
@foobarrio thanks, but it's ok for now! there's still a bunch more to be done with this rewrite. If you'd like to help, I am happy to have you bring you up to speed and help you contribute ❤️

Reply:
Oooh! I think you could help create our base annotation package! That would be very helpful :)

Reply:
I tried to `tsc` build and it balks on being unable to import Display is all.

Happy to help w/ whatever--point me in the right direction!